### PR TITLE
Add longer, more searchable package descriptions

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: packages/url_launcher/pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: url_launcher
-
+description: Flutter plugin for launching a URL on Android and iOS. Supports
+web, phone,SMS, and email schemes.
```
Comment (mravn-google) [inline packages/url_launcher/pubspec.yaml]: Add space after ,

File: packages/firebase_messaging/pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: firebase_messaging
-
-description: Firebase Cloud Messaging plugin for Flutter applications.
+description: Flutter plugin for Firebase Cloud Messaging, a cross-platform
+messaging solution that lets you reliably deliver messages at no cost.
```
Comment (mravn-google) [inline packages/firebase_messaging/pubspec.yaml]: Currently no cost. We may want to be careful here.

File: packages/firebase_analytics/pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: firebase_analytics
-
-description: Firebase Analytics plugin for Flutter.
+description: Flutter plugin for Firebase Analytics, an app measurement
```
Comment (mravn-google) [inline packages/firebase_analytics/pubspec.yaml]: This description doesn't mention Android and iOS

File: packages/android_intent/pubspec.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 name: android_intent
-description: A plugin for launching Intents on Android.
+description: Flutter plugin for launching Android Intents. Not supported on iOS.
```
Comment (mravn-google) [inline packages/android_intent/pubspec.yaml]: We mention iOS here, even though it is not supported. Is that what we want?

Comment (mit-mit) [reply packages/android_intent/pubspec.yaml]: I think it's OK to be explicit in this case

**Pull request merged**